code/_onclick/hud/hud.dm

```diff
@@ -232,14 +232,15 @@
 				if(mymob.client)
 					mymob.client.screen |= held // just to make sure it's visible post-login
 
-	var/hand_x_offset = -(world.icon_size/2)
-	for(var/i = 1 to length(swaphand_hud_objects))
-		var/obj/swap_elem = swaphand_hud_objects[i]
-		swap_elem.screen_loc = "CENTER:[hand_x_offset],BOTTOM:[hand_y_offset]"
-		if(i > 1) // first two elems share a slot
-			hand_x_offset += world.icon_size
-		if(mymob.client)
-			mymob.client.screen |= swap_elem
+	if(length(swaphand_hud_objects))
```

Comment:
this `if(length(swaphand_hud_objects))` check isn't strictly necessary because `for (var/i in 1 to 0)` will skip the loop body.